package.json

```
@@ -1,7 +1,7 @@
 {
-  "name": "strapi-provider-email-nodemailer",
-  "version": "2.0.0",
-  "description": "Nodemailer provider for Strapi with attacments support for latest Strapi versions and NTLM auth support",
+  "name": "strapi-provider-email-nodemailer-v3",
```

```suggestion
  "name": "strapi-provider-email-nodemailer",
```
@yutikom are you going to release a new major version? The `-v3` prefix would obviously need to be removed.